cluster/gce/util.sh

```diff
@@ -833,6 +844,9 @@ function create-subnetworks() {
         else
           echo "${color_yellow}Using pre-existing network ${NETWORK}, subnets won't be expanded to /19!${color_norm}"
         fi
+      elif [[ "${CREATE_CUSTOM_NETWORK:-}" == "true" ]]; then
+        assert-node-ip-range
```

Comment:
So `NODE_IP_RANGE` was previously only for IP Aliases, but now it's needed for CREATE_CUSTOM_NETWORK. Could we use the same default setting call https://github.com/kubernetes/kubernetes/blob/b5fbd71bbc6dc491a5d322baa1e28082d29559b3/cluster/gce/config-default.sh#L238 for the custom network case?

Reply:
That is what will happen in test, since we always set it in config-test.sh:
https://github.com/kubernetes/kubernetes/blob/0c1dcb01c57037410da5c5696ffec37549b0538f/cluster/gce/config-test.sh#L99

Does it make sense to do the same thing for config-default.sh?

Reply:
Done.